### PR TITLE
Make navigation much snappier

### PR DESCRIPTION
**Pull request by romanowski**

### CODE REVIEW

File: scala3doc/resources/dotty_res/styles/scalastyle.css [hunk 1]
```diff
@@ -159,63 +153,88 @@ th {
   width: 116px;
 }
 
-.sideMenuPart {
-  padding-left: 1em;
+/* Navigation */
+#sideMenu2 {
```
Comment (BarkingBad) [inline scala3doc/resources/dotty_res/styles/scalastyle.css:157]: Is there any clash with `sideMenu` name, that we use `sideMenu2`?

**Review by BarkingBad — COMMENTED** [review]
Looks OK

File: scala3doc/resources/dotty_res/styles/scalastyle.css (hunk 1 above)
Comment (BarkingBad) [reply scala3doc/resources/dotty_res/styles/scalastyle.css:157]: I assume you remove old `sideMenu` styles and elements

Comment (romanowski) [reply scala3doc/resources/dotty_res/styles/scalastyle.css:157]: yes

**Pull request merged**